dask_ml/decomposition/pca.py

```diff
@@ -175,13 +197,16 @@ class PCA(sklearn.decomposition.PCA):
       ``dask.linalg.svd_compressed``.
     * n_components : ``n_components='mle'`` is not allowed.
       Fractional ``n_components`` between 0 and 1 is not allowed.
+    * center : defaults to ``True`` and enables control over whether centering
+      gets implicitly performed as part of the PCA model steps.
     """
 
     def __init__(
         self,
         n_components=None,
         copy=True,
         whiten=False,
+        center=True,
```

Comment:
similar comment to `IncrementalPCA`...putting a new argument in the middle of the signature is a breaking change for anyone code constructing instance of this class with positional arguments. Please move this to the end of the signature.